### PR TITLE
Support downloading CurseForge modpack when USE_MODPACK_START_SCRIPT is false

### PR DESCRIPTION
**Pull request by Megasware128**
Not the best at bash scripts but I thought this is a clean way to get this done

### CODE REVIEW

File: scripts/start-deployCF [hunk 1]
```diff
@@ -35,10 +35,38 @@ FTB_SERVER_MOD=${FTB_SERVER_MOD:-$CF_SERVER_MOD}
...
+  fi
+  FTB_SERVER_MOD=${srv_modpack}
+}
+
 if ! isTrue "${USE_MODPACK_START_SCRIPT:-true}"; then
   if ! [ -f "${FTB_SERVER_MOD}" ]; then
-    log "ERROR unable to find requested modpack file ${FTB_SERVER_MOD}"
-    exit 2
+    downloadModpack
   fi
```
Comment (itzg) [inline scripts/start-deployCF:70]: I'm thinking the conditional can be taken away and this if-block just replaced with a call to `downloadMockpack`. That function is doing URL and file-existence checks.

**Review by itzg — CHANGES_REQUESTED** [review]
Nice refactoring! I think there's a little simplification can be done thanks to your work so far.

File: scripts/start-deployCF (hunk 1 above)
Comment (Megasware128) [reply scripts/start-deployCF]: Yeah you're right. Just tested it with a mounted volume zip, still works

**Review by itzg — APPROVED** [review]
Thanks!

**Pull request merged**